### DISCUSSION TITLE
Adicionando o Flunt no projeto do curso de Domínios Ricos

### DISCUSSION THREAD

**Discussion by richallyson** [balta-io/2801#76] [Perguntas e Respostas] [answered]
Salve Balta. Tava eu lá fazendo o curso de Domínios Ricos, e chegou o momento de adicionar o Flunt no projeto. Instalei bonitinho nas pastas, e quando fui rodar o projeto, apareceu essa mensagem:

Não foi possível localizar um projeto para executar. Verifique se existe um projeto em C:\Dev\01PaymentContext ou passe o caminho para o projeto usando --project.

Eu dei uma pesquisada, e vi que o projeto foi corrompido, de alguma forma que não sei bem ainda. E aconteceu depois de adicionar o Flunt ao projeto, antes disso estava rodando perfeitamente. Até agora eu não achei uma forma de recuperar o projeto kkkkk Deletei a solução antiga, fiz todo o processo de adicionar os csproj de domain, shared e tests, e mesmo assim, o projeto não roda. Dei restore, build, tentei restore pelo nuget mas ele diz que não reconhece o termo nuget (mesmo com o package manager dele instalado), reinstalei o visual studio code, e nada. Tentei procurar também nos fórum da vida e nada também. 

Uma coisa que eu percebi é que no processo de adicionar o flunt as pastas, sempre que você adicionava aparecia essa mensagem
![WhatsApp Image 2022-05-12 at 16 25 40](https://user-images.githubusercontent.com/24733986/168153536-0a4a813f-2f0a-4c88-b7d4-cc2c7375cddf.jpeg)

E no output essa:
![WhatsApp Image 2022-05-12 at 16 26 37](https://user-images.githubusercontent.com/24733986/168153573-8cf5f40b-3545-48bf-b858-566bd2d39823.jpeg)

Já no meu não acontece nada. Acho que justamente por não dar restore nessas dependências não resolvidas, meu projeto crashou e n rodou mais.

**Comment by andrebaltieri** [comment, marked as answer]
@richallyson vou pedir por favor que abra sua dúvida no canal do curso 1975 => https://github.com/balta-io/1975/discussions
Acho que já tem a resposta lá inclusive 💜

**Reply by richallyson** [reply]
Beleza. Procurei lá mas não tem a resposta pra a minha duvida.
